src/components/Button/README.md

```diff
@@ -187,6 +187,18 @@ Use to render a buttons that look like links where matching the surrounding text
 />
 ```
 
+### Plain destructive button
+
+<!-- example-for: online-store-ui -->
+
+Use to render a buttons that look like links where matching the surrounding text color is visually more appropriate. For example in the InlineError component.
```

Comment:
Here, I wouldn't mention the inline error component because using a plain button inline there would make the button indistinguishable. We recommend using plain monochrome buttons for error actions.

```suggestion
Use for actions that will delete merchant data or be otherwise difficult to recover from. Since they're less prominent, use for less important or less commonly used destructive actions. For example, plain buttons are used as actions in cards.
```